workbench-core/accounts/src/services/costCenterService.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { GetItemCommandOutput } from '@aws-sdk/client-dynamodb';
+import { AwsService, resourceTypeToKey, uuidWithLowercasePrefix } from '@aws/workbench-core-base';
+import Boom from '@hapi/boom';
+import Account from '../models/account';
+import CostCenter from '../models/costCenter';
+import CreateCostCenter from '../models/createCostCenter';
+import AccountService from './accountService';
+
+export default class CostCenterService {
+  private _aws: AwsService;
+  private readonly _tableName: string;
+
+  public constructor(constants: { TABLE_NAME: string }) {
+    const { TABLE_NAME } = constants;
+    this._tableName = TABLE_NAME;
+    this._aws = new AwsService({ region: process.env.AWS_REGION!, ddbTableName: TABLE_NAME });
+  }
+
+  public async getCostCenter(costCenterId: string): Promise<CostCenter> {
+    // Get by id
+    const response = (await this._aws.helpers.ddb
+      .get({
+        pk: costCenterId,
+        sk: costCenterId
+      })
+      .execute()) as GetItemCommandOutput;
+
+    if (response.Item === undefined) {
+      throw Boom.notFound(`Could not find cost center ${costCenterId}`);
+    }
+
+    console.log(`before ${JSON.stringify(response.Item)}`);
+
+    delete response.Item.pk;
+    delete response.Item.sk;
+    response.Item.accountId = response.Item.dependency;
```

Comment:
Why do we need to do line 41? Shouldn't the `Item.accountId` = `Item.dependency` already because the dynamo attributes will be the same?

Reply:
because we only have `accountId` at the application layer. Its `dependency` in the database layer